### PR TITLE
Fix issue #24: Use JSON for msl_teams.csv file enhancement 

### LINKED ISSUE #24: Use JSON for msl_teams.csv file
All other config files use JSON. Adapt the teams config file to also use the JSON format.
[https://github.com/RoboCup-MSL/RefBox2015/blob/master/mslrb2015/data/msl_teams.csv](url)

### PR DESCRIPTION
**Pull request by CriminalK**
Fixes #24

### CODE REVIEW

File: mslrb2015/data/msl_teams.json [hunk 1]
```diff
@@ -0,0 +1,422 @@
...
+	},
+	{
+		"longame24": "Mixed teams",
+		"UnicastAddr": "172.16.100.*",
+		"Team": "Mixed teams",
+		"shortname8": "Mixed te",
+		"MulticastAddr": "224.16.32.100"
+	},
+	{
+		"longame24": "TEST",
```
Comment (CriminalK) [inline mslrb2015/data/msl_teams.json]: Revert the change of the longame.

Comment (CriminalK) [inline mslrb2015/data/msl_teams.json]:
```suggestion
		"longame24": "LOCALHOST",
```

File: mslrb2015/TeamTableBuilder.pde [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    for (String settingsName: teamTableSettingsNames) {
+      row.setString(settingsName, teamSetting.getString(settingsName));
+    }
+  }
+  
+  Table build() {
+    Table table = new Table();
+
+    for (int i = 0; i < teamSettings.size(); i++) {
+      TableRow newRow = table.addRow();
```
Comment (jorritolthuis) [inline mslrb2015/TeamTableBuilder.pde:32]: The `makeTable()` is method is not used.
```suggestion
      TableRow newRow = makeTable();
```

**Review by jorritolthuis — CHANGES_REQUESTED** [review]

**Review by jorritolthuis — APPROVED** [review]
